.github/workflows/release.yaml

```diff
@@ -7,8 +7,36 @@ on:
     paths:
       - 'releases/CHANGELOG*.md'
 
+env:
+  LINUX_NODE_IMAGE_SUBSCRIPTION_ID: ${{ secrets.TEST_AZURE_SUB_ID }}
+  LINUX_NODE_IMAGE_RESOURCE_GROUP: ${{vars.LINUX_NODE_IMAGE_RESOURCE_GROUP}}
+  LINUX_NODE_IMAGE_GALLERY: ${{vars.LINUX_NODE_IMAGE_GALLERY}}
+  LINUX_NODE_IMAGE_NAME: ${{vars.LINUX_NODE_IMAGE_NAME}}
+  LINUX_NODE_IMAGE_VERSION: 2023.079.1
+  WINDOWS_NODE_IMAGE_SUBSCRIPTION_ID: ${{ secrets.TEST_AZURE_SUB_ID }}
+  WINDOWS_NODE_IMAGE_RESOURCE_GROUP: ${{vars.WINDOWS_NODE_IMAGE_RESOURCE_GROUP}}
+  WINDOWS_NODE_IMAGE_GALLERY: ${{vars.WINDOWS_NODE_IMAGE_GALLERY}}
+  WINDOWS_NODE_IMAGE_NAME: ${{vars.WINDOWS_NODE_IMAGE_NAME}}
+  WINDOWS_NODE_IMAGE_VERSION: 2023.079.1
+
 jobs:
+  sig_image_versions:
+    runs-on: ubuntu-latest
+    outputs:
+      LINUX_NODE_IMAGE_VERSION: ${{ env.LINUX_NODE_IMAGE_VERSION }}
+      WINDOWS_NODE_IMAGE_VERSION: ${{ env.WINDOWS_NODE_IMAGE_VERSION }}
+    steps:
+      - run: echo "Exposing sig image version variables"
+  validate-no-egress:
+    needs: sig_image_versions
+    uses: ./.github/workflows/test-vhd-no-egress.yaml
+    with:
+      from_branch: ''
+      LINUX_NODE_IMAGE_VERSION: ${{ needs.sig_image_versions.outputs.LINUX_NODE_IMAGE_VERSION }}
+      WINDOWS_NODE_IMAGE_VERSION: ${{ needs.sig_image_versions.outputs.WINDOWS_NODE_IMAGE_VERSION }}

```

Comment:
Environment variables are not extended to the reusable workflow it calls, so cannot directly use ${{ env.LINUX_NODE_IMAGE_VERSION }}. Further explanation of the solution here: https://github.com/orgs/community/discussions/26671